test/modes_internal_test.c

```diff
@@ -125,7 +125,6 @@ static AES_KEY *cts128_decrypt_key_schedule()
 
 typedef struct {
     const char *case_name;
-    int num;
     size_t (*transform_output)(const unsigned char *in, unsigned char *out,
                                size_t len);
     size_t (*encrypt_block)(const unsigned char *in,
```

Comment:
On side note it would be more appropriate to call next structure member, encrypt, for encrypt_stream. Because difference between the two is that this one uses single-block mode-agnostic subroutine and next one mode-specific CBC multi-block subroutine.